### PR TITLE
Feature/csi 5610 add an option to enable/disable call home

### PR DESCRIPTION
**Pull request by matancarmeli7**

### CODE REVIEW

File: controllers/tests/controller_server/utils_test.py [hunk 1]
```diff
@@ -534,3 +534,16 @@ def test_validate_parameters_match_volume_prefix_success(self):
         self._test_validate_parameters_match_volume(volume_field="name", volume_value="prefix_vol",
                                                     parameter_field=controller_config.PARAMETERS_VOLUME_NAME_PREFIX,
                                                     parameter_value="prefix")
+
+    def test_is_call_home_enabled_true(self):
+        self._test_is_call_home_enabled('true', True)
+
+    def test_is_call_home_enabled_false(self):
```
Comment (ArbelNathan) [inline controllers/tests/controller_server/utils_test.py:541]: add empty?

Comment (ArbelNathan) [inline controllers/tests/controller_server/utils_test.py:541]: add test for empty `get_env_return_value`?

**Review by ArbelNathan — DISMISSED** [review]
LGTM

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: controllers/servers/utils.py [hunk 2]
```diff
@@ -846,3 +847,7 @@ def get_replication_object_type_and_id_info(request):
             raise ValidationException(messages.UNSUPPORTED_REPLICATION_SOURCE_TYPE_MESSAGE)
     object_id_info = get_object_id_info(object_id, object_type)
     return object_type, object_id_info
+
+
+def is_call_home_enabled():
+    return getenv(servers_settings.ENABLE_CALL_HOME_ENV_VAR, 'true') == 'true'
```
Comment (leonid-s-usov) [inline controllers/servers/utils.py:853]: Does this mean that call home will be disabled by default? I couldn't find where this ENV is set

Comment (matancarmeli7) [reply controllers/servers/utils.py:853]: it's enabled by default, it comes from the [operator](https://github.com/IBM/ibm-block-csi-operator/pull/342)

Comment (leonid-s-usov) [reply controllers/servers/utils.py:853]: OK, thanks, [commented](https://github.com/IBM/ibm-block-csi-operator/pull/342#issuecomment-1592887227) there
